dockstore-webservice/src/main/java/io/dockstore/webservice/resources/MetadataResource.java

```diff
@@ -274,7 +274,7 @@ private String getRSS() {
         schema = @Schema(implementation = String.class)))
     @ApiOperation(value = "Returns the file containing runner dependencies.", response = String.class)
     public Response getRunnerDependencies(
-            @Parameter(name = "client_version", description = "The Dockstore client version")
+            @Parameter(name = "client_version", description = "The Dockstore client version", schema = @Schema(pattern = "(^(\\d+)\\.(\\d+)\\.(\\d+)$)|(^development-build$)"))
```

Comment:
Could you please update the description to include an example and/or say the expected format? Until looking at your PR, I would have assumed `1.12` would have been valid input for this call. Even though you now have an error message, it would still be useful.